### PR TITLE
Select package page

### PR DESCRIPTION
**Pull request by Peartes**
Bringing back the select a package page with some CSS cleanup.

- [x]  Keep only the Basic Package and center it
- [x]  It should say: *Use the open-source technology created by PrimeDAO & Balancer to host your own seed or LBP*
- [x]  Can you please bring back this page as given below?
- [x]  Please make sure we aren't using em in the css
- [x]  CSS is properly hierarchically structured
- [x]  if using flex we're going it correctly
- [x]  Remove Prime module completely
- [x]  Make sure that the launch page's BG is also displayed on the smaller viewport width

image

### CODE REVIEW

File: src/selectPackage/selectPackage.html [hunk 1]
```diff
@@ -1,32 +1,40 @@
 <template>
-  <require from="./packageContainer.html"></require>
   <div class="page animated-page au-animate selectPackage">
     <div class="banner">
       <div class="heading heading1">Select your package</div>
       <div class="subtext">
         <div>
-          PrimeDAO can help you host a succesfull permissionless token launch.
+          PrimeDAO can help you host a succesfull permissionless token launch. Select the package that firts best for your project.
```
Comment (dkent600) [inline src/selectPackage/selectPackage.html:7]: firts => works

File: src/home/home.html [hunk 2]
```diff
@@ -20,7 +20,7 @@
           </div>
         </div>
         <div class="buttonContainer">
-          <div class="launch" click.delegate="navigate('launch')">
+          <div class="launch" click.delegate="navigate('selectPackage')">
```
Comment (dkent600) [inline src/home/home.html:23]: Actually, this change should be reverted.  It is the launch page link that needs to go to selectPackage.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/home/home.html (hunk 2 above)
Comment (Peartes) [reply src/home/home.html:23]: Yes true, reverting

File: src/selectPackage/selectPackage.html [hunk 3]
```diff
@@ -1,32 +1,40 @@
 <template>
-  <require from="./packageContainer.html"></require>
   <div class="page animated-page au-animate selectPackage">
     <div class="banner">
       <div class="heading heading1">Select your package</div>
       <div class="subtext">
         <div>
-          PrimeDAO can help you host a succesfull permissionless token launch.
+          PrimeDAO can help you host a succesfull permissionless token launch. Select the package that works best for your project.
```
Comment (dkent600) [inline src/selectPackage/selectPackage.html:7]: Sorry I missed this the first review,  "succesfull" should be "successful"

**Review by dkent600 — CHANGES_REQUESTED** [review]
Per the mock, this page should have a background image .  Maybe is the same as the Launch page.

**Review by dkent600 — APPROVED** [review]

**Pull request merged**